arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -404,43 +404,43 @@
       }
     },
     "EndpointRangeDescription": {
-      "description": "Endpoint range description",
+      "description": "Port range details",
       "properties": {
         "startPort": {
-          "description": "Endpoint start port",
+          "description": "Starting port of a range of ports",
           "type": "integer"
         },
         "endPort": {
-          "description": "Endpoint end port",
+          "description": "End port of a range of ports",
           "type": "integer"
         }
       }
     },
     "NodeTypeDescription": {
-      "description": "NodeType description",
+      "description": "NodeType details",
```

Comment:
please elaborate the description - e.g. "Describes a node type in the cluster, each node type represents sub set of nodes in the cluster"